#!/usr/bin/env python
PACKAGE = "two_d_guidance"
    
from dynamic_reconfigure.parameter_generator_catkin import *
    
gen = ParameterGenerator()

gen.add("k_odom",       double_t, 0, "Odometry Gain", 0.85,  0.1,   2.)
gen.add("k_lm",       double_t, 0, "Landmark gain", 0.075,  0.03,   0.3)

exit(gen.generate(PACKAGE, "two_d_guidance", "trr_state_estimator"))
